packages/react/src/tabs/Tabs.tsx

```diff
@@ -8,15 +8,15 @@ import * as styles from "./Tabs.css";
 
 type TabsProps = BoxProps<
   typeof RadixTabs.Root,
-  { appearance?: "primary" | "secondary" }
+  { appearance?: "primary" | "secondary"; justified?: boolean }
```

Comment:
let's do multiple lines here
```suggestion
  {
    appearance?: "primary" | "secondary";
    justified?: boolean;
  }
```

Reply:
also was justified the name we finalized on? i thought it was something like `equalWidth`

IMO this should be automatic if possible based on the current screen size

so if possible let's do it in CSS - otherwise do it based on media query - but let's not expose this prop to the consumers